### DISCUSSION TITLE
Perhaps something to fix regarding NetFx472WebAsPrereq

### DISCUSSION THREAD

**Discussion by mvollmer1** [wixtoolset/issues#7750] [Questions]
In my bundle.wxs I have the following as the first definition in the chain.

	    <PackageGroupRef Id="NetFx472WebAsPrereq" /> 

During the install, the log shows the following two messages...

[1A00:1888][2023-09-21T16:55:36]i000: AdjustForDetectedPackages package id:NetFx472Web has state:Present 
[1A00:1888][2023-09-21T16:55:36]i000: This package is uknown:NetFx472Web  Fix this problem.

The problem is not impeding any of my work that I can tell.  Thought someone might want to look into this after they sort their sock drawer.
Windows Server 2016 has .Net Framework 4.8 installed already during my testing.

If you need anything else, or have questions, I'm happy to help.

**Comment by barnson** [comment]
Those messages aren't coming from WiX.
